apps/desktop/src/components/v3/MultiStackView.svelte

```diff
@@ -117,7 +121,7 @@
 	});
 
 	// Throttle calls to the reordering code in order to save some cpu cycles.
-	const throttledDragOver = throttle(onReorderDragOver, 250);
```

Comment:
Making it 25 instead of 250 makes it much more responsive and the performance is still  good